WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -182,6 +182,9 @@ public WpkhDescriptors GetOutputDescriptors(string password, Network network)
 	[JsonProperty(PropertyName = "AutoCoinJoin")]
 	public bool AutoCoinJoin { get; set; } = DefaultAutoCoinjoin;
 
+	[JsonProperty(PropertyName = "OutputWalletName")]
+	public string? OutputWalletName { get; set; }
```

Comment:
The wallet should forget about this after the restart (meeting agreement @BTCparadigm @MaxHillebrand ). That means you do not need to write this into the file.

Reply:
done